Packs/Pcysys/pack_metadata.json

```diff
@@ -1,18 +1,16 @@
 {
     "name": "Pentera",
-    "description": "An Integration with Pentera by Pcysys",
+    "description": "Automate remediation actions based on Pentera, the Automated Security Validation Platform, proactively exposing high-risk vulnerabilities.",
     "support": "partner",
-    "currentVersion": "1.0.4",
-    "author": "Pcysys",
+    "currentVersion": "1.1.0",
+    "author": "Pentera",
     "url": "",
-    "email": "support@pcysys.com",
+    "email": "support@pentera.io",
     "created": "2020-03-10T08:37:18Z",
     "categories": [],
     "tags": [],
     "useCases": [],
     "keywords": [],
-    "githubUser": [
-        "TalPcysys"
-    ],
-    "certification": "certified"
-}
\ No newline at end of file
+    "certification": "certified",
+    "authorImage": "content/Packs/Pcysys/Integrations/Pcysys/Pcysys_image.png"
```

Comment:
Do they have a new Github user they can add?

Reply:
If no user, it will default to support email address no @Bargenish ? Lets do that.